docs/libssh2_sftp_unlink_ex.3

```diff
@@ -9,7 +9,7 @@ libssh2_sftp_unlink_ex - unlink an SFTP file
 #include <libssh2_sftp.h>
 
 int
-libssh2_sftp_unlink_ex(LIBSSH2_SFTP *sftp, const char *filename, unsigned int filename_len);
+libssh2_sftp_unlink_ex(LIBSSH2_SFTP *sftp, const char *filename, size_t filename_len);
```

Comment:
We'd want that definitely, but that looks like an ABI break. We can either postpone it to a future "ABI break" release or add `libssh2_sftp_unlink_ex2` or similar.

Reply:
I'm honestly not sure why this change was part of this commit in the Panic repo. I can revert it so it doesn't break ABI.

Reply:
Yeah, I agree to keep it separate. It's a good change, we should probably keep track of it in `docs/TODO`'s `At next SONAME bump` section to remember.